src/fontra/backends/designspace.py

```diff
@@ -62,6 +62,9 @@
 from .filewatcher import Change, FileWatcher
 from .ufo_utils import extractGlyphNameAndCodePoints
 
+# from glyphsLib.builder.custom_params import KNOWN_PARAM_HANDLERS
```

Comment:
We don't have a hard dependency on glyphsLib anymore, and also this would break the abstraction boundary, so this is not an option.

Reply:
Thanks for the info. I removed it. Closing this as resolved.